src/shared/model/connection/index.ts

```diff
@@ -83,6 +83,7 @@ class ConnectionStateStore {
 
     try {
       await penumbra.disconnect();
+      window.location.reload();
```

Comment:
suggestion: we want to ensure the page reload is handled in the `finally` block, independent of whether the disconnection succeeds or fails.

```ts
    try {
      await penumbra.disconnect();
    } catch (error) {
      console.error("Error during disconnect:", error);
    } finally {
      window.location.reload();
    }
```

consider for instance when the extension's connection context becomes invalidated, resulting in an unsuccessful disconnection.

https://github.com/user-attachments/assets/7f846f0e-2df3-467d-bfe2-ae6444c20812

Reply:
Good point, fixed.